src/AccountListItem.cpp

```diff
@@ -34,36 +34,37 @@ AccountListItem::DrawItem(BView* owner, BRect frame, bool complete)
 	if (IsSelected()) {
 		if (IsEnabled()) {
 			if (fAccount->IsClosed()) {
-				owner->SetHighColor(GetColor(BC_SELECTION_NOFOCUS));
-				owner->SetLowColor(GetColor(BC_SELECTION_NOFOCUS));
+				owner->SetHighUIColor(B_LIST_SELECTED_BACKGROUND_COLOR, B_DISABLED_MARK_TINT);
+				owner->SetLowUIColor(B_LIST_SELECTED_ITEM_TEXT_COLOR, B_DISABLED_MARK_TINT);
 			} else {
-				owner->SetHighColor(GetColor(BC_SELECTION_FOCUS));
-				owner->SetLowColor(GetColor(BC_SELECTION_FOCUS));
+				owner->SetHighUIColor(B_LIST_SELECTED_BACKGROUND_COLOR);
+				owner->SetLowUIColor(B_LIST_SELECTED_ITEM_TEXT_COLOR);
 			}
 		} else {
-			owner->SetHighColor(230, 230, 230);
-			owner->SetLowColor(230, 230, 230);
+			owner->SetHighUIColor(B_LIST_SELECTED_BACKGROUND_COLOR, B_DISABLED_MARK_TINT);
+			owner->SetLowUIColor(B_LIST_SELECTED_ITEM_TEXT_COLOR, B_DISABLED_MARK_TINT);
 		}
 	} else {
 		if (fAccount->IsClosed()) {
-			owner->SetHighColor(240, 240, 240, 128);
-			owner->SetLowColor(240, 240, 240, 128);
+			owner->SetHighUIColor(B_LIST_BACKGROUND_COLOR, B_DISABLED_MARK_TINT);
+			owner->SetLowUIColor(B_LIST_ITEM_TEXT_COLOR, B_DISABLED_MARK_TINT);
 		} else {
-			owner->SetHighColor(255, 255, 255, 128);
-			owner->SetLowColor(255, 255, 255, 128);
+			owner->SetHighUIColor(B_LIST_BACKGROUND_COLOR);
+			owner->SetLowUIColor(B_LIST_ITEM_TEXT_COLOR);
 		}
 	}
 	owner->FillRect(frame);
```

Comment:
Something isn't right here. We go through all the effort to determine the right High- and LowColors, depending on selection, enabled and account-closed. Only to draw a rectangle.
Then in line 59 we potentially change the HighColor to stroke the frame.
Then we change the HighColor again in line 64 to draw the text.

I think the text drawing should be done right after filling the rect, when the colours to use are still valid.
Drawing the rect only needs the HighColor set. Maybe it can be simplified by using rgb_color variables and set them to either selected/unselected and then tint them if needed (enabled/disabled, account-closed).

This may be applicable to other drawing code as well...

Currently, the drawing isn't correct. Try using  a white on blue text for selected list items:
![screenshot](https://github.com/HaikuArchives/CapitalBe/assets/6107158/1080d4cc-8244-4afb-9759-1dd295c44eff)